src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: var(--primary-text-color);
+
+  --accent-color: #00acdc;
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  max-width: 100%;
+  height: auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+  margin: 0 0 4px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star.svg) no-repeat center;
+  width: 16px;
+  height: 16px;
+  border-radius: 1px;
+}
+
+.stars :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__rate-text {
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: center;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-text-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__btn-buy {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  line-height: 16px;
+  padding: 12px 0;
+  border: 1px solid var(--accent-color);
+  border-radius: 5px;
+  background: #fff;
+  color: var(--accent-color);
+}
+
+.card__btn-buy:hover {
+  background-color: var(--accent-color);
+  color: #fff;
```

Comment:
Your button have to be blue without hover and become white with hover, swap them.